src/awkward/_v2/operations/convert/ak_from_parquet.py

```diff
@@ -70,9 +70,6 @@ def from_parquet(
             storage_options,
             row_groups,
             columns,
-            max_gap,
-            max_block,
-            footer_sample_size,
```

Comment:
Users will sometimes get a warning that the `footer_sample_size` is too small. That's fine, but it would be annoying if there is no way to set the `footer_sample_size` and silence the warning.

Reply:
This is not used, because fsspec.parquet doesn't offer any particular speedup for just reading the footer. The code just uses fs.open now.